tests/RequestHandlerTest.php

```diff
@@ -23,7 +23,7 @@ public function testRequestThrowsErrorOnMalformedBaseUrl()
 
         $options = array('some kinda option');
 
-        $this->expectException(\GuzzleHttp\Exception\ConnectException::class);
+        $this->expectException(\GuzzleHttp\Exception\TransferException::class);
```

Comment:
Newer version of Guzzle throws a different exception: this is the common parent.